### DISCUSSION TITLE
swipe-controls toggle between what action is performed when long press swiped

### DISCUSSION THREAD

**Discussion by LoV432** [ReVanced/revanced-discussions#541] [Question]
Instead of having option to long press swipe for volume/brightness gestures OR always doing volume/brightness gestures, the options should  be:
(
Long press swipe for volume/brightness gestures - Normal swipe for native Youtube function
OR
Long press swipe for native Youtube function - Normal swipe for volume/brightness gestures
)
i do have it working ish [here](https://github.com/LoV432/revanced-integrations/tree/reverse-swipe)

**Comment by oSumAtrIX** [comment]
Can you make a PR?
